packages/zwave-js/src/lib/commandclass/SceneControllerConfigurationCC.ts

```diff
@@ -0,0 +1,490 @@
+			endpoint: this.endpoint.index,
+			groupId,
+			sceneId,
+			dimmingDuration,
+		});
+
+		await this.driver.sendCommand(cc, this.commandOptions);
+	}
+
+	public async getLastActivated(): Promise<
+		| {
+				groupId: number;
+				sceneId: number;
+				dimmingDuration: Duration;
+		  }
+		| undefined
+	> {
+		this.assertSupportsCommand(
+			SceneControllerConfigurationCommand,
+			SceneControllerConfigurationCommand.Get,
+		);
+		return this.get(0);
+	}
+
+	public async get(
+		groupId: number,
+	): Promise<
+		| {
+				groupId: number;
+				sceneId: number;
+				dimmingDuration: Duration;
+		  }
+		| undefined
+	> {
+		this.assertSupportsCommand(
+			SceneControllerConfigurationCommand,
+			SceneControllerConfigurationCommand.Get,
+		);
+
+		const cc = new SceneControllerConfigurationCCGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			groupId,
+		});
+		const response = await this.driver.sendCommand<SceneControllerConfigurationCCReport>(
+			cc,
+			this.commandOptions,
+		);
+
+		// Return value includes "groupId", because if get(0) is called
+		// the returned report will include the actual groupId of the
+		// last activated groupId / sceneId
+		if (response)
+			return pick(response, ["groupId", "sceneId", "dimmingDuration"]);
+	}
+}
+
+@commandClass(CommandClasses["Scene Controller Configuration"])
+@implementedVersion(1)
+export class SceneControllerConfigurationCC extends CommandClass {
+	declare ccCommand: SceneControllerConfigurationCommand;
+
+	public determineRequiredCCInterviews(): readonly CommandClasses[] {
+		// AssociationCC is required and MUST be interviewed
+		// before SceneControllerConfigurationCC to supply groupCount
+		return [
+			...super.determineRequiredCCInterviews(),
+			CommandClasses.Association,
+		];
+	}
+
+	public async interview(complete: boolean = true): Promise<void> {
+		const node = this.getNode()!;
+		const endpoint = this.getEndpoint()!;
+		const api = endpoint.commandClasses[
+			"Scene Controller Configuration"
+		].withOptions({
+			priority: MessagePriority.NodeQuery,
+		});
+
+		this.driver.controllerLog.logNode(node.id, {
+			message: `${this.constructor.name}: doing a ${
+				complete ? "complete" : "partial"
+			} interview...`,
+			direction: "none",
+		});
+
+		const groupCount = this.getGroupCountCached();
+		if (groupCount === 0) {
+			this.driver.controllerLog.logNode(node.id, {
+				endpoint: this.endpointIndex,
+				message: `soft failing interview (missing association group count)`,
+				direction: "none",
+			});
+
+			this.interviewComplete = false;
+			return;
+		}
+
+		this.driver.controllerLog.logNode(node.id, {
+			endpoint: this.endpointIndex,
+			message: `supports ${groupCount} association groups`,
+			direction: "none",
+		});
+
+		// Always query scene and dimmer duration for each association group
+		// skipping group #1, which is reserved for Lifeline
+		for (let groupId = 2; groupId <= groupCount; groupId++) {
```

Comment:
See other comment regarding group 1, I think it would be good to have this information.